src/platform/esp32/main-esp32.cpp

```diff
@@ -169,9 +169,10 @@ void esp32Setup()
 // #define APP_WATCHDOG_SECS 45
 #define APP_WATCHDOG_SECS 90
 
-#ifdef CONFIG_IDF_TARGET_ESP32C6
```

Comment:
Those are changes from the new ESP-IDF - rather han gating to the Processor or device type, can we check and compare the ESP-IDF version?